tests/test_field_handler.py

```diff
@@ -6,7 +6,7 @@
                              HasManyDescriptor, HasAndBelongsToManyDescriptor)
 from remodel.utils import create_tables, create_indexes
 
-from . import BaseTestCase, DbBaseTestCase
```

Comment:
Hmm, that's weird. I've run your changes with the relative import there and it worked fine. I'll have a better look at it.